### PR TITLE
Updated Documentation navigation

### PR DESCRIPTION
**Pull request by mattray**
Reworked the sidebar navigation to logically group pages and added the OpenCost Specification and Community pages.Check the Vercel preview to see the new navigation.

### CODE REVIEW

**Comment by mattray** [comment]
The OpenCost Specification page includes the updated spec in https://github.com/opencost/opencost/pull/2039

File: docs/FAQ.md [hunk 1]
```diff
@@ -8,35 +8,35 @@ sidebar_position: 3
...
+You can [install](installation/install) the project in minutes.
 
 ### Do I need to pay to use OpenCost?
 
 No. OpenCost is open source (Apache 2.0) and free to use.
 
 ### How does OpenCost relate to Kubecost?
 
-The OpenCost implementation is the cost allocation engine originally built by Kubecost. This implementation is actively used in all versions of Kubecost for building a cost allocation model. Founding OpenCost members have announced intentions to onboard OpenCost as a CNCF sandbox project. If the CNCF accepts our onboarding application, the CNCF will become the neutral home for OpenCost moving forward and it will be moved from the Kubecost Github organization.
+The OpenCost implementation is the cost allocation engine originally built by Kubecost. This implementation is actively used in all versions of Kubecost for building a cost allocation model, to which they add additional features on top of in their commercial offering. OpenCost is a CNCF sandbox project, so it is free to use and open to contributions from all community members.
```
Comment (skl) [inline docs/FAQ.md:23]:
```suggestion
The OpenCost implementation is the cost allocation engine originally built by Kubecost. This implementation is actively used in all versions of Kubecost for building a cost allocation model, to which they add additional features in their commercial offering. OpenCost is a CNCF sandbox project, so it is free to use and open to contributions from all community members.
```

File: docs/installation/helm.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 3
+---
+# Managing with Helm
+
+There is a [community-supported Helm chart](https://github.com/opencost/opencost-helm-chart) with extensive customization settings exposed via the [values.yaml](https://github.com/opencost/opencost-helm-chart/blob/main/values.yaml). The Helm installation still assumes an existing [Prometheus](prometheus) installation. The following commands work with the https://github.com/opencost/opencost-helm-chart repository checked-out locally and modified the `values.yaml` with your own local changes (alternatively you may keep them in a separate, source-controlled file).
```
Comment (skl) [inline docs/installation/helm.md:6]:
```suggestion
There is a [community-supported Helm chart](https://github.com/opencost/opencost-helm-chart) with extensive customization settings exposed via the [values.yaml](https://github.com/opencost/opencost-helm-chart/blob/main/charts/opencost/values.yaml). The Helm installation still assumes an existing [Prometheus](prometheus) installation. The following commands work with the [opencost/opencost-helm-chart](https://github.com/opencost/opencost-helm-chart) repository checked-out locally and with the `values.yaml` modified for your own local changes (alternatively you may keep them in a separate, source-controlled file).
```

File: docs/installation/helm.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+### Upgrading OpenCost with Helm
+
+```
+helm upgrade opencost . --namespace opencost -f values.yaml
+```
+
+### Sidegrading with Helm
+
+If you wish to override the version of OpenCost installed by the Helm chart (downgrading or alternate versions), update the `values.yaml` to set the tag of the downloaded image (you can also change the registry and repository if necessary).
```
Comment (skl) [inline docs/installation/helm.md:22]:
```suggestion
If you wish to override the version of OpenCost installed by the Helm chart (e.g. downgrading or alternative versions), update the `values.yaml` to set the tag of the downloaded image (you can also change the registry and repository if necessary).
```

File: docs/installation/prometheus.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+If you are going to connect existing Prometheus instance which is already consuming KSM metrics, please consider visiting this page about [KSM metrics emission](https://docs.kubecost.com/architecture/ksm-metrics) because OpenCost currently implements the same architecture and you might get overlapping metrics.
+
+At this point you can proceed to [installing OpenCost](install#install-opencost).
+
+### Providing your own Prometheus
+
+If you want to use your own Prometheus:
+ 1. Set the PROMETHEUS_SERVER_ENDPOINT [environment variable](https://github.com/opencost/opencost/blob/develop/kubernetes/opencost.yaml#L154) to the address of your Prometheus server.
```
Comment (skl) [inline docs/installation/prometheus.md:25]:
```suggestion
 1. Set the `PROMETHEUS_SERVER_ENDPOINT` [environment variable](https://github.com/opencost/opencost/blob/eadd55d5fcee528547492df4baf29497dc019470/kubernetes/opencost.yaml#L154) to the address of your Prometheus server.
```

File: docs/integrations/integrations.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+# OpenCost Integrations
+
+OpenCost may be integrated with a wide variety of technologies. Below is a list of known OpenCost integrations.
+
+## Cloud Integrations
+* [Azure](http://aka.ms/aks/OpenCost-AKS)
```
Comment (skl) [inline docs/integrations/integrations.md:9]: Not sure if you'd want to link to the Azure page instead maybe?

```suggestion
* [Azure](../configuration/azure-prices)
```

File: docs/integrations/integrations.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+## Cost metric services
+* [Prometheus](../installation/prometheus)
+* [Cortex](https://cortexmetrics.io/)
+* [Thanos](https://thanos.io/)
+* [Victoria Metrics](https://victoriametrics.com/)
+
+## Cost visualization
+* [Grafana](https://grafana.com/orgs/kubecost)
+* [Grafana Labs](https://grafana.com/blog/2023/06/26/rein-in-spending-with-kubernetes-cost-monitoring-in-grafana-cloud/)
```
Comment (skl) [inline docs/integrations/integrations.md:23]:
```suggestion
* [Grafana Cloud](https://grafana.com/blog/2023/06/26/rein-in-spending-with-kubernetes-cost-monitoring-in-grafana-cloud/)
```

File: docs/integrations/kubectl-cost.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
 ---
-sidebar_position: 5
+sidebar_position: 3
 ---
 
-# Kubectl cost queries
+# kubectl cost
 
-`kubectl-cost` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) that provides easy CLI access to Kubernetes cost allocation metrics via the [OpenCost API](./api.md). It allows developers, devops, and others to quickly determine the cost & efficiency for any Kubernetes workload.
+`kubectl-cost` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) that provides easy CLI access to Kubernetes cost allocation metrics via the [OpenCost API](/api). It allows developers, devops, and others to quickly determine the cost & efficiency for any Kubernetes workload.
```
Comment (skl) [inline docs/integrations/kubectl-cost.md:7]:
```suggestion
`kubectl-cost` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) that provides easy CLI access to Kubernetes cost allocation metrics via the [OpenCost API](/api). It allows developers, devops, and others to quickly determine the cost and efficiency for any Kubernetes workload.
```

File: docs/integrations/integrations.md (hunk 5 above)
Comment (mattray) [reply docs/integrations/integrations.md:9]: I thought the Azure blog post had a lot of other context, but it makes sense to keep things internal when they're available.

**Comment by mattray** [comment]
Will close https://github.com/opencost/opencost-website/issues/52

**Review by skl — APPROVED** [review]

**Pull request merged**